WeakAuras/HelperFunctions.lua

```diff
@@ -0,0 +1,39 @@
+WeakAuras.helperFunction = {
+  -- Unit Aura functions that return info about the first Aura matching the spellName or spellID given on the unit. 
+  GetUnitAura = function(unit, spell, filter)
+      for i = 1, 40 do
+          local name, _, _, _, _, _, _, _, _, spellId = UnitAura(unit, i, filter)
+          if not name then return end
+          if spell == spellId or spell == name then
+              return UnitAura(unit, i, filter)
+          end
+      end
+  end
+  ,
+  GetUnitBuff = function(unit, spell, filter)
+      return GetUnitAura(unit, spell, filter)
```

Comment:
Ensure that the filter either doesn't exist or contains HELPFUL. Also, GetUnitAura here refers to a nonexistent global. Instead, use `return WeakAuras.helperFunction.GetUnitAura`.